### PR TITLE
Updated Website-header-css edges-hovers, buttons, margins

### PR DESCRIPTION
**Pull request by viditkothari**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -31,36 +31,21 @@
...
-            <a href="">FAQ & Info</a>
-          </li>
-          <li>
-            <a href="">Contact</a>
-          </li>
+          <li><a href="">About</a></li>
+          <li><a href="">Why donate?</a></li>
+          <li><a href="">Eligibility</a></li>
+          <li class="brand"><a><img src="/assets/images/logo.png" alt=""></a></li>
+          <li><a href="">How to donate?</a></li>
```
Comment (utkarsh2102) [inline public/index.html]: Sweet.
Hope this does not overlap like the previous one.

**Comment by viditkothari** [comment]
I'll need a dollar

Best,
Vidit

File: public/index.html [hunk 2]
```diff
@@ -226,8 +185,8 @@ <h5>Stay Connected</h5>
           <!--Column1-->
           <div class="footer-pad">
             <div class="footer-subscribe">
-              <h5>YOU MAY NOT THINK YOU'RE AN ACTIVE DONOR, BUT WE DO.</h5>
-
+              <h5>YOU MAY NOT THINK YOU'RE AN ACTIVIST...<br/>BUT WE DO</h5>
```
Comment (utkarsh2102) [inline public/index.html]: Hahaha, was this necessary? :sweat_smile:

File: public/assets/css/main.css [hunk 3]
```diff
@@ -26,50 +26,63 @@ nav {
...
     min-width: 100%;
     color: #fff;
     font-family: Spectral, serif;
-    font-size: 21px;
+    font-size: 1.2em;
 }
 
-nab .brand {}
+nav a:hover {
+    color: #fff;
```
Comment (utkarsh2102) [inline public/assets/css/main.css]: Sweet.

**Pull request merged**